recipes/edyn/0.x.x/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.8)
+project(test_package CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+include(${CMAKE_BINARY_DIR}/conan_paths.cmake OPTIONAL)
+conan_basic_setup()
+
+find_package(EnTT REQUIRED)
```

Comment:
You don't need to look for entt. conan will add this dependency automatically for you.
```suggestion
```